sigmf/archive.py

```diff
@@ -1,22 +1,8 @@
-# Copyright 2021 GNU Radio Foundation
+# Copyright 2021, 2022 Multiple Authors
```

Comment:
For the year do we need to keep appending? Can we just make it 2023 and keep it clean?

Reply:
Not really sure actually... a quick google search seems to indicate that the first year of publication is all that is necessary.

Reply:
Including a year actually hasn't been necessary since 1989, but everyone does it anyway.

In short, I don't care, and it doesn't really matter, so do whatever you like / is easiest 🤣

Reply:
`Copyright: Multiple Authors` it is